tests/Jenkins/utilities.sh

```diff
@@ -894,6 +894,32 @@ dropDBs(){
   python "${TESTCODE}/DIRAC/tests/Jenkins/dirac-drop-db.py" "${dbs[@]}" "${DEBUG}"
 }
 
+#-------------------------------------------------------------------------------
+# diracOptimizers:
+#
+#   installs all optimizers on the file executors
+#
+#-------------------------------------------------------------------------------
+
+diracOptimizers(){
+  echo '==> [diracOptimizers]'
+
+  local executors=$(cat executors | grep WorkloadManagementSystem | cut -d ' ' -f 2 | grep -v Base)
+  for executor in $executors
+  do
+    echo "==> calling dirac-cfg-add-option executor $executor"
+    python "${TESTCODE}/DIRAC/tests/Jenkins/dirac-cfg-add-option.py" "executor" "$executor"
+  done
+
+  echo "==> calling dirac-cfg-add-option executor Optimizers"
+  python "${TESTCODE}/DIRAC/tests/Jenkins/dirac-cfg-add-option.py" "executor" "WorkloadManagementSystem/Optimizers"
+  if ! dirac-executor WorkloadManagement/Optimizers --cfg ServerInstallDIR/diracos/etc/Production.cfg "${DEBUG}"
```

Comment:
It would be better to use instead the `dirac-install-component` similar to what is done in the `diracServices` function.